packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -549,7 +629,14 @@ class OffsetLayer extends ContainerLayer {
   ///
   /// The [offset] property must be non-null before the compositing phase of the
   /// pipeline.
-  Offset offset;
+  Offset get offset => _offset;
+  set offset(Offset offset) {
```

Comment:
(here and elsewhere) we generally prefer the name `value` for the argument to the setter, it makes for easier copy/paste later. (see the style guide)